releases/dogwood/3/fun/config/cms/docker_run_production.py

```diff
@@ -167,53 +167,36 @@
 CACHE_REDIS_HOST = config("CACHE_REDIS_HOST", default="redis")
 CACHE_REDIS_PORT = config("CACHE_REDIS_PORT", default=6379, formatter=int)
 CACHE_REDIS_DB = config("CACHE_REDIS_DB", default=1, formatter=int)
-CACHE_REDIS_URI = "redis://{}:{}/{}".format(CACHE_REDIS_HOST, CACHE_REDIS_PORT, CACHE_REDIS_DB)
 CACHE_REDIS_BACKEND = config("CACHE_REDIS_BACKEND", default="django_redis.cache.RedisCache")
+CACHE_REDIS_URI = "redis://{}:{}/{}".format(CACHE_REDIS_HOST, CACHE_REDIS_PORT, CACHE_REDIS_DB)
 CACHE_REDIS_CLIENT = config("CACHE_REDIS_CLIENT", default="django_redis.client.DefaultClient")
 
+CACHES_DEFAULT_CONFIG = {"BACKEND": CACHE_REDIS_BACKEND, "LOCATION": CACHE_REDIS_URI,
```

Comment:
[PEP8] Wouldn't it be cleaner to write that on multiple lines like so:

```suggestion
CACHES_DEFAULT_CONFIG = {
    "BACKEND": CACHE_REDIS_BACKEND,
    "LOCATION": CACHE_REDIS_URI,
    "OPTIONS": { "CLIENT_CLASS": CACHE_REDIS_CLIENT }
}
```

Reply:
That's right, thanks ! I forgot to run [black](https://github.com/psf/black) on the files.
I'll fix that.

Reply:
Done :heavy_check_mark: